langchain4j-core/src/main/java/dev/langchain4j/rag/content/Content.java

```diff
@@ -63,4 +73,9 @@ public static Content from(String text) {
     public static Content from(TextSegment textSegment) {
         return new Content(textSegment);
     }
+
+    public static Content from(EmbeddingMatch<TextSegment> embeddingMatch) {
```

Comment:
`Content` is store-agnostic, can we keep it so (not to couple it with embeddings)?

Reply:
I agree, just made another commit for this